corehq/form_processor/models.py

```diff
@@ -426,7 +426,7 @@ def read_content(self, stream=False):
             if not self.blob_bucket and '-' in str(self.attachment_id):
                 try:
                     # http://manage.dimagi.com/default.asp?239638
-                    blob = db.get(self.blob_id, self.blobdb_bucket(True))
+                    blob = db.get(self.blob_id, self.blobdb_bucket(remove_dashes=False))
                 except (KeyError, NotFound, BadName):
```

Comment:
To get rid of this `try`/`except` we will need a migration for attachments that were broken by the new UUID field type up until this PR gets deployed, correct?